modules/navier_stokes/test/tests/finite_volume/wcns/enthalpy_equation/enthalpy_equation.i

```diff
@@ -0,0 +1,335 @@
+H = 0.015 #halfwidth of the channel, 10 cm of channel height
+L = 3
+bulk_u = 0.01
+p_ref = 101325.0
+#inlet_temp = 860.
+#flux = 4000.0
+
+advected_interp_method = 'upwind'
+
+[Mesh]
+  [gen]
+    type = GeneratedMeshGenerator
+    dim = 2
+    xmin = 0
+    xmax = ${L}
+    ymin = -${H}
+    ymax = ${H}
+    nx = 100
+    ny = 30
```

Comment:
```suggestion
    nx = 20
    ny = 10
```

Reply:
Just the make the tests faster and less troublesome to regold.

Reply:
decreased the nx and ny